config.json

```diff
@@ -539,7 +539,14 @@
       "topics": [
 
       ]
-    }
+    },
+    {
+      "slug": "rotational-cipher",
+      "difficulty": ,
+      "topics": [
+
+      ]
+    },
```

Comment:
The comma on this line should be left out. This is what is causing the build to fail